pages/docs/concepts/asyncapi-document/structure.md

```diff
@@ -0,0 +1,432 @@
+---
+title: AsyncAPI Structure
+weight: 32
+---
+The structure of a AsyncAPI document can be defined as a specific format in which the document of the specification is to be written. An AsyncAPI document may be made up of a single document or be divided into multiple, connected parts at your discretion. In the latter case, Reference Objects are used. The structure of a AsyncAPI document has certain fields that need to be followed and implemented.
```

Comment:
Do you mean, we should have a unique description of AsyncAPI document to begin with then lead to the definition of what is the structure of a asyncapi document and so on ?

Reply:
Yes

Reply:
In that case, maybe I should wait for @alequetzalli to do that and incorporate the same or how do you suggest we move forward with this?

Reply:
For the record, and as commented in the call, it's up to you and @alequetzalli to decide how to proceed in order to keep consistency between docs.

Reply:
Heyooo 😄, great discussion point, so glad Sergio brought this up!

1. @TRohit20, Sergio is spot on in recommending we stick to one definition for `AsyncAPI document`, so that we don't end up with misc docs defining that term differently. 
2. That said, **this doc focuses on `document structure`,** _not_ introducing the `AsyncAPI document`. Thus, we must remove your sentences defining `AsyncAPI document` because that is not needed in this doc. _(Remember: we will define `AsyncAPI document` in the `introduction.md` doc that I am writing under https://github.com/asyncapi/website/pull/1868](https://github.com/asyncapi/website/pull/1868)_